tests/sklearn/test_sklearn_models.py

```diff
@@ -1575,6 +1575,7 @@ def test_fitted_compiled_error_raises(
     check_fitted_compiled_error_raises(model_class, n_bits, x, y)
 
 
+@pytest.mark.flaky
```

Comment:
same, please create an issue and add a FXIME here if this PR has detected a new flaky issue